RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Fix addition of undesirable white space in HTML
```

Comment:
You can update the release message as @justinmayer proposed in its (1) point.

Reply:
Done. Not sure it this is enough. I looked at the previous files and they usually have one line summaries so I tried to do the same. I can I more detailed description if required.

Reply:
It looks better! I would just add the root cause, for example: `Fix addition of undesirable white space in HTML because of BeautifulSoup.prettiy()`

Reply:
Thank you for the suggestion. Changed.